packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -2076,6 +2077,222 @@ export function subscribeToGestureDirection(
   }
 }
 
+type EventListenerOptionsOrUseCapture =
+  | boolean
+  | {
+      capture?: boolean,
+      once?: boolean,
+      passive?: boolean,
+      signal?: AbortSignal,
+      ...
+    };
+
+type StoredEventListener = {
+  type: string,
+  listener: EventListener,
+  optionsOrUseCapture: void | EventListenerOptionsOrUseCapture,
+};
+
+export type FragmentInstance = {
+  _fragmentFiber: Fiber,
+  _eventListeners: void | Array<StoredEventListener>,
+  addEventListener(
+    type: string,
+    listener: EventListener,
+    optionsOrUseCapture?: EventListenerOptionsOrUseCapture,
+  ): void,
+  removeEventListener(
+    type: string,
+    listener: EventListener,
+    optionsOrUseCapture?: EventListenerOptionsOrUseCapture,
+  ): void,
+  focus(): void,
+};
+
+function FragmentInstancePseudoElement(
```

Comment:
Nit: This is not really a "PseudoElement" in the DOM terminology. The reason `ViewTransitionPseudoElement` is called this is because it represents a reified/first-class version of a PseudoElement Instance as a DOM concept of an Element that's part of another Element.

This is similar but it's conceptually more like a real Element or just a Fragment.

So maybe FragmentInstance is the right name?

This matters for when you log these object in the console and the name of the instance shows as `[FragmentInstance]`.

Reply:
Yes just realized this as I started to read the view transition spec with the PsuedoElement concepts. Will rename to `FragmentInstance` and have the type be `FragmentInstanceType`